content/docs/notebooks/why-use-jupyter-notebook.md

```diff
@@ -1,15 +1,19 @@
 +++
-title = "Overview of Jupyter Notebooks in Kubeflow"
-description = "Why use Jupyter Notebooks in Kubeflow"
+title = "Overview of Jupyter notebooks in Kubeflow"
```

Comment:
Actually, the N should be capital here, and lower case in the description. The Kubeflow docs convention is to use title case in page titles.